Getting-Started.md

```diff
@@ -37,6 +37,87 @@ You will need to install ElasticSearch (6.1.0), Kibiter (6.1.4) and a MySQL/Mari
 You will have to install the GrimoireLab components listed above, and use the following docker-compose to have ElasticSearch, Kibiter and MariaDB.
 Note that you can omit the `mariadb` section in case you have MySQL/MariaDB already installed in your system.
 
+***
+
+#### Docker installation
+
+Installation for Ubuntu 
+
+[Reference for Docker installation](https://phoenixnap.com/kb/how-to-install-docker-on-ubuntu-18-04)
+
+```
+Sudo apt-get update
```

Comment:
Hi @pugnator-12 
can you change this to `sudo`. I think the capital 'S' might not work.

Reply:
Sorry for that, Thanks!